ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -306,6 +307,9 @@ const Annotations = React.memo(({ variant }) => {
           : <div>hg19: liftover failed</div>
         )
       }
+      {cpxIntervals && cpxIntervals.lenght > 0 &&
```

Comment:
`lenght` is a typo so this isn't working (which is why the screenshot isn't showing any intervals)

Reply:
Oh..., it does show, all `intervals` disappear. :(

Reply:
The layout for now:
![image](https://user-images.githubusercontent.com/33550059/120524470-17212380-c3a5-11eb-99bf-b026915bc9d8.png)

Reply:
There is no genotype record with a `sample_id` of `BEG_1078_2_01` in the response from Elasticsearch.
The `_source` field in the response is shown below.
```
{
  'sortedTranscriptConsequences': [
    {
      'gene_symbol': 'SNTG2',
      'gene_id': 'ENSG00000172554',
      'major_consequence': 'NEAREST_TSS'
    }
  ],
  'xpos': 2000849376,
  'sv_callset_Hom': 13,
  'start': 849376,
  'genotypes': [
    {
      'num_alt': 0,
      'sample_id': 'BEG_0992-3_01',
      'gq': 999,
      'cn': 2
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_1025-1_1',
      'gq': 796,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_1230-3_01',
      'gq': 840,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_14-4_1',
      'gq': 99,
      'cn': 6
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_851-3_1',
      'gq': 442,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_887-2_1',
      'gq': 148,
      'cn': 5
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_887-3_1',
      'gq': 847,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_0761-2_01',
      'gq': 473,
      'cn': 6
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_0761-3_01',
      'gq': 425,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_0992-2_01',
      'gq': 331,
      'cn': 5
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_1025-3_1',
      'gq': 935,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_1078-1_01',
      'gq': 999,
      'cn': 6
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_1078-3_01',
      'gq': 94,
      'cn': 1
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_1230-2_01',
      'gq': 999,
      'cn': 2
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_21-1_1',
      'gq': 999,
      'cn': 3
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_21-3_1',
      'gq': 142,
      'cn': 7
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_21-4_1',
      'gq': 425,
      'cn': 5
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_851-1_1',
      'gq': 849,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_851-2_1',
      'gq': 888,
      'cn': 1
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_894-1_1',
      'gq': 999,
      'cn': 3
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_894-2_1',
      'gq': 142,
      'cn': 6
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_916-3_1',
      'gq': 1,
      'cn': 6
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_0761-1_01',
      'gq': 999,
      'cn': 2
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_0992-1_01',
      'gq': 999,
      'cn': 2
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_1025-2_1',
      'gq': 999,
      'cn': 2
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_1230-1_01',
      'gq': 468,
      'cn': 1
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_14-5_1',
      'gq': 559,
      'cn': 7
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_887-1_1',
      'gq': 194,
      'cn': 5
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_894-3_1',
      'gq': 1,
      'cn': 5
    },
    {
      'num_alt': 1,
      'sample_id': 'BEG_916-1_1',
      'gq': 11,
      'cn': 5
    },
    {
      'num_alt': 0,
      'sample_id': 'BEG_916-2_1',
      'gq': 999,
      'cn': 2
    }
  ],
  'samples_num_alt_2': [],
  'samples_num_alt_1': [
    'BEG_1025-1_1',
    'BEG_1230-3_01',
    'BEG_14-4_1',
    'BEG_851-3_1',
    'BEG_887-2_1',
    'BEG_887-3_1',
    'BEG_0761-2_01',
    'BEG_0761-3_01',
    'BEG_0992-2_01',
    'BEG_1025-3_1',
    'BEG_1078-1_01',
    'BEG_1078-3_01',
    'BEG_21-3_1',
    'BEG_21-4_1',
    'BEG_851-1_1',
    'BEG_851-2_1',
    'BEG_894-2_1',
    'BEG_916-3_1',
    'BEG_1230-1_01',
    'BEG_14-5_1',
    'BEG_887-1_1',
    'BEG_894-3_1',
    'BEG_916-1_1'
  ],
  'contig': '2',
  'cpx_intervals': [
    {
      'start': 849216,
      'end': 849396,
      'type': 'DUP',
      'chrom': '2'
    }
  ],
  'sc': 513,
  'sv_type_detail': 'dDUP',
  'sf': 0.359244,
  'svType': 'CPX',
  'end': 850065,
  'sn': 1428,
  'variantId': 'CMG.phase1_CMG_CPX_chr2_1'
}
```